src/guc.c

```diff
@@ -486,6 +487,24 @@ _guc_init(void)
 							   /* assign_hook= */ NULL,
 							   /* show_hook= */ NULL);
 #endif
+
+	DefineCustomIntVariable("timescaledb.default_replication_factor",
```

Comment:
I suggest changing to the following to make clear that this is a hypertable setting.

```suggestion
	DefineCustomIntVariable("timescaledb.hypertable_replication_factor_default",
```

Then we'd also have a similar GUC, as discussed:

```sql
timescaledb.hypertable_distributed_default
```